### ISSUE TITLE
Add documentation for DB based ResourceGroup Managers

### ISSUE THREAD

**Issue by phd3** [prestosql/presto#4450]
Currently we have it only for file-based resource group management. It may be a good idea to add an example with the table schemas. 

ref: 
https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/admin/resource-groups.rst
https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/admin/resource-groups-example.json

**Comment by mosabua** [comment]
Sounds good. Do you want to submit a PR yourself?

**Comment by mosabua** [comment]
Ping @phd3 ?

**Comment by phd3** [comment]
@mosabua yup, will address PR comments soon (unless someone else wants to pick it up).
